main_server.py

```
@@ -1000,6 +1178,616 @@ async def api_key_settings(request: Request):
+            "error": "Steamworks未初始化",
+            "message": "请确保Steam客户端已运行且已登录"
+        }, status_code=503)
+    
+    try:
+        # 转换item_id为整数
+        item_id_int = int(item_id)
+        
+        # 获取物品状态
+        item_state = steamworks.Workshop.GetItemState(item_id_int)
+        
+        # 创建查询请求，传入必要的published_file_ids参数
```

Ruff (0.14.5)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)